lib/chef/azure/commands/enable.rb

```diff
@@ -66,9 +66,9 @@ def enable_chef
     begin
       configure_chef_only_once
 
-      install_chef_service if @exit_code == 0
+      install_chef_service if @exit_code == 0 && windows?

```

Comment:
@mwrock The `install_chef_service` method is for the `Windows` platform and the `enable_chef_service` method is for the `Linux` platform. At the very first time during the `chef extension` installation, these methods are called (after the first `bootstrap` is invoked and the `node registration` is done with the `chef server`) to install the `chef service` on the respective platform. The methods internally contains the checks to see if the service is running or not and then to take appropriate action.